demo/code_component/run.py

```diff
@@ -0,0 +1,19 @@
+import gradio as gr
+
+css = (
+    "footer {display: none !important;} .gradio-container {min-height: 0px !important;}"
+)
+
+with gr.Blocks(css=css) as demo:
+    gr.Code(
+        value="""
+    def hello_world():
+        return "Hello, world!"
+    
+    print(hello_world())
+    """,
```

Comment:
The code is showing up with extra spaces for me: 

<img width="386" alt="image" src="https://user-images.githubusercontent.com/1778297/226476219-5c5e6c9c-216c-47ef-b5cc-1b2244c93325.png">

Reply:
I actually think we should modify the preprocessing function of `Code` to strip this, similar to how we do for `Markdown`. Will create an issue

Reply:
Created an issue here: https://github.com/gradio-app/gradio/issues/3552 but in the meantime we can just manually remove from here

Reply:
Good catch! 

This is what it looks like now

![image](https://user-images.githubusercontent.com/41651716/226484623-3b2cdb40-1bd0-4f4b-8ca4-13227af3c891.png)

Removed the label because of #3467 and made it interactive!